tools/bowtie2.wdl

```diff
@@ -0,0 +1,410 @@
+        write_paired_concordant: "Write pairs that aligned concordantly at least once to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit unset and either the 0x40 or 0x80 bit set (depending on whether it's mate #1 or #2)."
+        quiet: "Print nothing besides alignments and serious errors."
+        metrics_file: "Write bowtie2 alignment metrics to file. Useful for debugging."
+        metrics_stderr: "Write  bowtie2 alignment metrics to STDERR. Not mutually exclusive with `metrics_file`."
+        metrics_interval: "Report internal counters & metrics every N seconds. Only used if `metrics_file` and/or `metrics_stderr` is specified."
+        no_unal: "Suppress SAM records for unaligned reads."
+        no_head: "Suppress header lines, i.e. lines starting with @."
+        no_sq: "Suppress @SQ header lines."
+        read_group: "Read group record to include in output SAM/BAM header"
+        addl_rg_text: "add <text> ('label:value') to @RG line of SAM header"
+        omit_sec_seq: "When printing secondary alignments, Bowtie 2 by default will write out the SEQ and QUAL strings. Specifying this option causes Bowtie 2 to print an asterisk in those fields instead."
+        sam_no_quane_trunc: "Suppress standard behavior of truncating readname at first whitespace at the expense of generating non-standard SAM."
+        xeq: "Use '='/'X', instead of 'M,' to specify matches/mismatches in SAM record"
+        soft_clipped_unmapped_tlen: "Exclude soft-clipped bases when reporting TLEN (template length). Only used if `end_to_end` is false."
+        sam_append_comment: "Append FASTA/FASTQ comment to SAM record, where a comment is everything after the first space in the read name."
+        sam_opt_config: "Toggle optional SAM fields. Prefix fields with `-` to turn off. Example: '-MD,YP,-AS' will disable the `MD` and `AS` fields and enable the `YP` field."
+        threads: "Number of threads to use for alignment. Threads will synchronize when parsing reads and outputting alignments. Searching for alignments is highly parallel, and speedup is close to linear. Increasing threads increases Bowtie 2's memory footprint."
+        reorder: {
+            description: "Force SAM output order to match order of input reads",
+            help: "Guarantees that output SAM records are printed in an order corresponding to the order of the reads in the original input file, even when `threads` is set greater than 1. Specifying `reorder` and setting `threads` greater than 1 causes Bowtie 2 to run somewhat slower and use somewhat more memory than if `reorder` were not specified. Has no effect if `threads` is set to 1, since output order will naturally correspond to input order in that case.",
+        }
+        memory_map: "use memory-mapped I/O for index; many 'bowtie's can share"
```

Comment:
This is meaningless in WDL context, no?

Reply:
With containerization, I think so. If the processes ran natively on the host and multiple jobs landed on the same host, then it would work (I think).